website/views/layouts/layout.ejs

```diff
@@ -171,7 +171,7 @@
               <iframe src="//ghbtns.com/github-btn.html?user=fleetdm&amp;repo=fleet&amp;type=watch&amp;count=true"
               allowtransparency="true" frameborder="0" scrolling="0" width="100" height="20"></iframe>
             </span>
-            <a style="padding: 4px 16px; line-height: 24px; color: #FFFFFF" class="btn btn-sm btn-primary align-items-center d-flex" href="/get-started">Try it out</a>
+            <a style="padding: 4px 16px; line-height: 24px; color: #FFFFFF" class="btn btn-sm btn-primary align-items-center d-flex" href="/get-started" onClick="ga('send','event','[Fleetdm]','[Buttons]','[try it top]');">Try it out</a>
```

Comment:
I wonder if it will be tracked successfully since clicking the link will also begin navigating away?  @timmy-k might have experience here-- otherwise we should do some research.

Reply:
Are we using Universal Analytics? If so this is the broad implementation that we’d use, but I haven’t looked at specifics. https://support.google.com/analytics/answer/1136920?hl=en#zippy=%2Cin-this-article

Pretty sure it’s a solvable problem though.